src/model/CustomMenuHolder.php

```diff
@@ -128,6 +130,15 @@ function requireDefaultRecords()
             }
         }
 
+        if (!$config->Menus()->exists()) {
```

Comment:
We have a `CustomMenusMigrationTask` called below, I think it would make more sense to migrate this logic into the task? It can probably be merged into the existing loop?